internal/router/inventory.go

```diff
@@ -59,7 +59,13 @@ func (a *app) ClassificationHandler(ctx *gin.Context) {
 	case "page-content":
 		ctx.HTML(http.StatusOK, "", pages.InventoryClassificationContent(params))
 	case "sidenav-page-wrapper":
-		ctx.HTML(http.StatusOK, "", pages.InventoryClassificationContentWithSideNav(params))
+		if htmx.HasMatchingPath(ctx) {
+			htmx.Retarget(ctx, "#page-content")
+			htmx.Reswap(ctx, "innerHTML")
+			ctx.HTML(http.StatusOK, "", pages.InventoryClassificationContent(params))
+		} else {
+			ctx.HTML(http.StatusOK, "", pages.InventoryClassificationContentWithSideNav(params))
+		}
```

Comment:
Add this if/else to this route handler and the below MonsterFamilyHandler to make it so sidenav anchor tag requests to the same page will return just the page content instead of MainContentWithSidenav.